_sass/fix-analytics/section/risk-and-pricing.scss

```diff
@@ -0,0 +1,4 @@
+.pricing__row {

```

Comment:
Ser ud til at vi kan klarer det her med flex-utility funktioner i stedet for at have specifik pricing styling.